Src/cmor.c

```diff
@@ -6145,7 +6145,9 @@ int cmor_CreateFromTemplate(int nVarRefTblID, char *templateSTH,
 /*      or an internal attribute.  Otherwise we just skip it and the    */
 /*      user get a warning.                                             */
 /* -------------------------------------------------------------------- */
-        } else if (strcmp(szToken, GLOBAL_ATT_CONVENTIONS) == 0) {
+        } else if (strcmp(szToken, GLOBAL_ATT_CONVENTIONS) == 0
+                    || strcmp(szToken, GLOBAL_ATT_LICENSE_ID) == 0
+                    || strcmp(szToken, GLOBAL_ATT_LICENSE_TYPE) == 0) {
```

Comment:
Should we also have `license_url` defined here, seems to be defined below as an internal variable